core/src/main/java/com/sequenceiq/cloudbreak/service/kerberos/KerberosConfigService.java

```diff
@@ -35,10 +35,14 @@
     private CloudbreakRestRequestThreadLocalService restRequestThreadLocalService;
 
     @Override
-    protected EnvironmentResourceRepository<KerberosConfig, Long> repository() {
+    protected WorkspaceResourceRepository<KerberosConfig, Long> repository() {
         return repository;
     }
 
+    @Override
+    protected void prepareDeletion(KerberosConfig resource) {
```

Comment:
This method has a proper implementation in the previously extended abstract class `AbstractEnvironmentAwareService`

Reply:
yes, but this class does not extend the `AbstractEnvironmentAwareService` anymore. But I'll make a proper one here also.